src/components/AccountContainer.js

```diff
@@ -42,20 +42,61 @@ class AccountContainer extends Component {
           category: "Coffee",
           amount: 365
         }
-      ]
+      ],
+      filteredTransactions: [{
+        id: 5,
+        posted_at: "neverhappened",
+        description: "fake",
+        category: "unreal",
+        amount: 40
+      }]
     }
+    this.handleChange = this.handleChange.bind(this)
+    this.filterTransactionsList = this.filterTransactionsList.bind(this)
   }
 
   handleChange(event) {
-    // your code here
+    this.setState({
+      searchTerm: event.target.value
+    })
+    this.filterTransactionsList(event.target.value)
+
   }
 
-  render() {
+  filterTransactionsList(query){
+
+    let temp =  this.state.transactions.filter(
+        trans => trans.description.toLowerCase().includes(query.toLowerCase()) || trans.category.toLowerCase().includes(query.toLowerCase()))
+
+    this.setState({
+      filteredTransactions: temp
+    })
+
+
+  }
 
+  isListFiltered(){
```

Comment:
the name of this method implies that it returns a boolean. I would call this `filteredTransactions`